drivers/bus/isapnp/hardware.c

```diff
@@ -20,13 +20,19 @@
 
 #endif /* UNIT_TEST */
 
+ULONG_PTR IsaConfigPorts[2] =
```

Comment:
Ideally these could stay `ULONG`, and the `(PUCHAR)IsaConfigPorts[i]` casts below could be converted to `UlongToPtr(IsaConfigPorts[i])`